KMM_Showcase/src/main/java/co/nimblehq/kmm/showcase/android/JobDetailsFragment.kt

```diff
@@ -18,5 +27,21 @@ class JobDetailsFragment : BaseFragment<FragmentJobDetailsBinding>() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
         setupToolbar(R.string.job_details_title)
+
+        // TODO Display response in integration task
+        mainScope.launch {
+            kotlin.runCatching {
+                repository.getJobDetails()
+            }.onSuccess {
+                binding.titleTv.text = it.message
+            }.onFailure {
+                binding.titleTv.text = it.localizedMessage
+            }
+        }
+    }
+
+    override fun onDestroy() {
+        super.onDestroy()
+        mainScope.cancel()
```

Comment:
Can we bring this method to the BaseFragment as well? Imagine in the future if we have a new fragment, we need to override it again, right? @hoangnguyen92dn